cri/v1alpha2/cri.go

```diff
@@ -683,6 +686,12 @@ func (c *CriManager) CreateContainer(ctx context.Context, r *runtime.CreateConta
 	sandboxConfig := r.GetSandboxConfig()
 	podSandboxID := r.GetPodSandboxId()
 
+	// get sandbox
+	sandbox, err := c.ContainerMgr.Get(ctx, podSandboxID)
```

Comment:
Is this part of code related to the hookplugin? @rudyfly Just wish confirmation from you.

In addition, the codes added gets instance `sandbox, but this `sandbox` is used in L741, about 50 lines of code away from here.

If you are not putting this code on purpose, I am wondering if we could locate the code just along with L741 to improve aggregation.

Reply:
Oh, I see you are constructing `sandboxMeta ` for the hook plugin. 

But also wish you could answer the second question.